rotkehlchen/tests/external_apis/test_coingecko.py

```diff
@@ -75,74 +70,6 @@ def test_coingecko_historical_price(session_coingecko):
     assert price == Price(FVal('2065.603754353392'))
 
 
-@pytest.mark.parametrize('use_clean_caching_directory', [True])
-def test_asset_icons_for_collections(icon_manager: IconManager) -> None:
```

Comment:
we need to be able to move relevant tests to rust. As this is no longer testable in python